.github/workflows/ci.yml

```diff
@@ -6,7 +6,53 @@ on:
   push:
     branches:
       - master
+
 jobs:
+  job1:
```

Comment:
Is this still a WIP?

Reply:
yeah yeah, do not worry about this one, this is purely using as an experiment